e2e/__tests__/nativeEsm.test.ts

```diff
@@ -21,7 +21,7 @@ test('test config is without transform', () => {
 });
 
 // The versions vm.Module was introduced
-onNodeVersions('^12.16.0 || >=13.2.0', () => {
+onNodeVersions('^12.16.0 || >=13.7.0', () => {
```

Comment:
why this change? The only thing you're changing in this PR is how to traverse FS looking for a file, that has nothing to do with what this test is testing

Reply:
escalade support native es modules via "exports" field. And it's broken in node < 13.7
https://github.com/lukeed/escalade/blob/master/package.json#L15-L23

See https://github.com/uuidjs/uuid/issues/436 https://github.com/ai/nanoid/issues/215

Reply:
But this test doesn't load escalade via ES imports. The native support talked about here is running a test with native ESM in it (using the `vm` APIs), `jest-resolve` runs outside of the test so that shouldn't matter.

Reply:
This is the point. Even commonjs usage is broken in <13.7 when "exports" field is used.

Reply:
ah, node broke CJS. Nice 😛 Maybe update the comment above though, since we don't check just `vm.Module` anoymore? No biggie either way